src/EMS/RingdownStatus.js

```diff
@@ -7,15 +7,28 @@ import Alert from '../Components/Alert';
 
 function RingdownStatus({ className, onStatusChange, ringdown }) {
   const [showCancel, setShowCancel] = useState(false);
+  const [confirmCancel, setShowConfirmCancel] = useState(false);
   const [showRedirect, setShowRedirect] = useState(false);
+  const [confirmRedirect, setConfirmRedirect] = useState(false);
 
   function handleCancel() {
     setShowCancel(false);
-    onStatusChange(ringdown, Ringdown.Status.CANCELLED);
+    setShowConfirmCancel(true);
   }
 
   function handleRedirect() {
     setShowRedirect(false);
+    setConfirmRedirect(true);
+    // onStatusChange(ringdown, Ringdown.Status.REDIRECTED);
```

Comment:
Are you sure we don't want to do the status change on this event? Because otherwise, right now, when you press "Confirm redirect patient" and you see the success popup that says "Hospital notified", it hasn't actually been changed and the hospital has NOT been notified yet, not until you choose one of the next buttons.